modules/zrtp/zrtp.c

```diff
@@ -198,15 +244,74 @@ static bool udp_helper_recv(struct sa *src, struct mbuf *mb, void *arg)
 }
 
 
+static void sig_hash_encode(struct zrtp_stream_t *stream,
+                         struct sdp_media *m)
+{
+	char buf[ZRTP_SIGN_ZRTP_HASH_LENGTH + 1];
+	zrtp_status_t s;
+	int err;
+
+	if (!use_sig_hash)
+		return;
```

Comment:
I think it is better to move this check to where the function is invoked.

then we also avoid the negated if statement